### ISSUE TITLE
Host Marionette long-form discussion on Discourse

### ISSUE THREAD

**Issue by jasonLaster** [marionettejs/backbone.marionette#2483]
### Types of discussions

1. Many important discussions occur in the github issue tracker (33 open, 135 closed)
https://github.com/marionettejs/backbone.marionette/issues?q=is%3Aopen+is%3Aissue+label%3Adiscussion

2. Many important discussions occur in the gitter main channel

### Github Issue Discussion
The issue tracker is not the best place to host discussions for a couple reasons:
+ discussions are long-lived and often not resolvable in their nature
+ discussions are more interesting to the larger community than the typical (bug)
+ discussions are less about implementation details and more "big picture"
+ discusssions keep us from realistically getting to issues to 0

### Gitter Discussions
There are several reasons why some large gitter discussions would be better in disqus:
+ discus would let each discussion happen in parrallel, some discussions by their nature are big topics that many people should weigh in on over time. chat makes it hard to do that because the conversation moves on.
+ the main channel is so active that it is hard to get a summary of the discussions. If big discussions occured in disqus, it'd be easier for people who wanted to check once a day to skim the active discussions and weigh in.

By the way, I've started working on setting up a disqus hosted site for Mn. It needs a domain and email provider, but otherwise good. If we use it, I'd move the discussions from github issues to disqus

Im sure there are plenty of more reasons for and against. Let discuss, ooh the irony...

**Comment by paulfalgout** [comment]
I'm :+1: on a better discussion solution..  I'm iffy on wanting another service.. but don't necessarily want to pick a bad one..

But what about gitter # channels.. that lets discussions happen in parallel as well.. the issue would then be how you'd get/maintain a list of those gitter channels.

**Comment by paulovieira** [comment]
@jasonLaster You meant "disqus" or "discourse"? 

I though disqus was meant only for blog comments.

I'm a big fan of discourse, and it keeps getting better and better. I have a discourse forum running on a 1gb digital ocean VPS for 6 months. No problems so far - http://forum.mubi.pt/

**Comment by jasonLaster** [comment]
Oops. Discourse

**Comment by paulfalgout** [comment]
Closing in favor of active https://github.com/marionettejs/backbone.marionette/issues/2910
